iis-wf/iis-wf-export-actionmanager/src/test/java/eu/dnetlib/iis/wf/export/actionmanager/sequencefile/FieldAccessor.java

```diff
@@ -0,0 +1,143 @@
+package eu.dnetlib.iis.wf.export.actionmanager.sequencefile;
+
+import java.lang.reflect.Array;
+import java.lang.reflect.Method;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.commons.lang.StringUtils;
+
+/**
+ * Accesses object fields using getter methods of field names encoded in properties file as
+ * keys. Nested field names are separated with dots.
+ * 
+ * Notice: field names should start with capital letters as they appear in getter method.
+ * 
+ * All fields requiring decoder should be marked with '$' prefix. 
+ * 
+ * @author mhorst
+ *
+ */
+public class FieldAccessor {
```

Comment:
`PropertyUtils` won't make this method significantly shorter so I have refactored this code into several specialized methods. To be committed...